src/state/opamStateConfig.ml

```diff
@@ -195,7 +201,11 @@ let initk k =
     ?no_env_notice:(E.noenvnotice ())
     ?locked:(E.locked () >>| function "" -> None | s -> Some s)
     ?no_depexts:(E.nodepexts ())
-
+    ?verbose_on:
+     (E.verbose () >>| fun s ->
+      OpamStd.String.split s ',' |>
+      List.map OpamPackage.Name.of_string |>
+      OpamPackage.Name.Set.of_list)
```

Comment:
It is better to add an environment variable converter in `OpamStd.Config` than splitting the string here. You can see how it is done for `OPAMDEBUGSECTIONS` with `OpamStd.Config.env_sections`, or more simple float variables with `OpamStd.Config.env_float`.
You can add a function that handles string with `,` separator, that converts it to a string list ; and convert that string list into names set here.